adodb-datadict.inc.php

```diff
@@ -907,7 +907,7 @@ function _getSize($ftype, $ty, $fsize, $fprec, $options=false)
 
 
 	// return string must begin with space
-	function _createSuffix($fname,&$ftype,$fnotnull,$fdefault,$fautoinc,$fconstraint,$funsigned)
```

Comment:
Can you please add a PHPDoc block ?

And the `// return string must begin with space` comment can be removed from all the inherited methods.

Reply:
Done.